### PR TITLE
Add doc for how to move a cluster

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:
Add a doc on how to move a cluster

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by jichenjc** [comment]
recheck

**Comment by jichenjc** [comment]
@hidekazuna @sbueringer take a look? The CI is not stable now (https://github.com/theopenlab/openlab/issues/581) but this is doc only change...

**Comment by hidekazuna** [comment]
@jichenjc Moving cluster is nothing special except for #626 . IMO we do not need documentation about moving once we resolve #626 .

**Comment by jichenjc** [comment]
well.. I think it's still helpful to give an end to end sample ..

**Comment by hidekazuna** [comment]
recheck

**Comment by hidekazuna** [comment]
@jichenjc Please update using `clusterctl.cluster.x-k8s.io/move`.

**Comment by jichenjc** [comment]
@hidekazuna updated, thanks~

File: docs/mover.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+kube-system                         etcd-kind-control-plane                                          1/1     Running   1          40m
+kube-system                         kindnet-ng2gf                                                    1/1     Running   0          40m
+kube-system                         kube-apiserver-kind-control-plane                                1/1     Running   1          40m
+kube-system                         kube-controller-manager-kind-control-plane                       1/1     Running   1          40m
+kube-system                         kube-proxy-h6rmz                                                 1/1     Running   0          40m
+kube-system                         kube-scheduler-kind-control-plane                                1/1     Running   1          40m
+local-path-storage                  local-path-provisioner-bd4bb6b75-ft7wh                           1/1     Running   0          40m
+```
+
+target cluster (using external cloud manager)
```
Comment (hidekazuna) [inline docs/mover.md:43]: Target cluster (Below is an example of external cloud provider)

File: docs/mover.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+kube-system   etcd-capi-openstack-control-plane-82xck                      1/1     Running   0          22m
+kube-system   kube-apiserver-capi-openstack-control-plane-82xck            1/1     Running   0          22m
+kube-system   kube-controller-manager-capi-openstack-control-plane-82xck   1/1     Running   2          22m
+kube-system   kube-proxy-4f9k8                                             1/1     Running   0          21m
+kube-system   kube-proxy-gjd55                                             1/1     Running   0          21m
+kube-system   kube-scheduler-capi-openstack-control-plane-82xck            1/1     Running   2          22m
+kube-system   openstack-cloud-controller-manager-z9jtc                     1/1     Running   1          4m9s
+```
+
+# Install openstack providers into target cluster
```
Comment (hidekazuna) [inline docs/mover.md:61]: Install OpenStack Cluster API provider into target cluster

**Review by hidekazuna — CHANGES_REQUESTED** [review]
Let's use "OpenStack", not "openstack".

**Comment by jichenjc** [comment]
@hidekazuna update. thanks

**Comment by jichenjc** [comment]
recheck

**Pull request merged**